proposals/NNNN-Restructuring-OnResetTimeout.md

```diff
@@ -0,0 +1,122 @@
+# Restructuring OnResetTimeout
+
+* Proposal: [SDL-NNNN](NNNN-Restructuring-OnResetTimeout.md)
+* Author: [Ankur Tiwari](https://github.com/ATIWARI9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is to use OnResetTimeout for applicable interfaces so as to update the function and expand it to be used by broader set of RPCs than just in _UI_ and _TTS_ interfaces.
```

Comment:
I think this should be updated to "...by a broader set of RPCs..." This should be updated in the PR description as well.